metering/metering-uninstall.adoc

```diff
@@ -1,13 +1,22 @@
 :context: metering-uninstall
 [id="metering-uninstall"]
-= Uninstalling Metering
+= Uninstalling metering
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can remove metering from your {product-title} cluster.
+Uninstall metering by removing the Metering Operator and associated metering components from your {product-title} cluster and uninstalling the Operator from your metering namespace.
 
-include::modules/metering-uninstall.adoc[leveloffset=+1]
+[NOTE]
+====
+Metering does not manage or delete Amazon S3 bucket data. You must manually clean up S3 buckets used to store metering data.
+====
+
+[id="metering-remove"]
+== Removing metering from your cluster
 
+You can remove the Metering Operator from your cluster by following the documentation on xref:../operators/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
```

Comment:
Do we need to mirror the note from the documentation this link references? Essentially you're deleting metering from OLM's management, which results in the deletion of the Metering Operator, but not any of the resources that Operator had provisioned. So basically, it would be:
1.) Delete the Metering Operator from the openshift-metering namespace from the list of installed operators. This deletes the actual Metering Operator, but the actual metering stack will stuck be able to run (as the operand Pods will still exist).
2.) Delete the MeteringConfig custom resource in the openshift-metering namespace to clean up resources that metering had installed itself.